build/pyinstaller/terminal.spec

```diff
@@ -75,6 +75,10 @@ added_files = [
     (os.path.join(pathex, "blib2to3", "PatternGrammar.txt"), "blib2to3"),
 ]
 
+
+if is_darwin and os.path.exists(os.path.join(pathex, "xgboost")):
+    added_files.append((os.path.join(pathex, "xgboost"), "xgboost"))
```

Comment:
should the hook replace this? @tehcoderer

Reply:
yes sir you can remove that cause it' run with this
![image](https://user-images.githubusercontent.com/61340027/213261805-96d5e9c9-7940-4e04-b5f5-d745025a6f46.png)